dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java

```diff
@@ -392,15 +393,17 @@ public Entry(long id) {
     @JsonIgnore
     public abstract String getEntryPath();
 
-    @JsonProperty(access = JsonProperty.Access.READ_ONLY)
-    public abstract EntryType getEntryType();
-
     @JsonProperty(access = JsonProperty.Access.READ_ONLY)
     public abstract EntryTypeMetadata getEntryTypeMetadata();
 
     @JsonIgnore
     public abstract boolean isHosted();
 
+    @JsonProperty(access = JsonProperty.Access.READ_ONLY)
```

Comment:
Noticed some repetition, so updated this. Should have no change in the UI as the info surfaced is the same